st2common/st2common/models/db/profiling.py

```diff
@@ -0,0 +1,105 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
since this is not a model can you move it to utils?

`st2common.models.utils` is likely a better home for profiling.py